packages/mdc-shape/README.md

```diff
@@ -65,6 +65,42 @@ Styles for applying shapes for specific corners such as drawer looks like this:
 
 Where, only top-right & bottom-right corners are customizable and it automatically flips radius values based on RTL context when `$rtl-reflexive` is set to true.
 
+### Global theming
```

Comment:
I'm very wary of enumerating components and overrides here because they will _undoubtedly_ end up out of sync the moment design makes us change something in the respective package and we forget to change it here. This seems like strictly design documentation, so I would push on designers that this should be something that gets documented on material.io so we can cross-link to it and they can update it directly.